src/java/org/apache/cassandra/service/paxos/PrepareVerbHandler.java

```diff
@@ -45,12 +46,21 @@ public void doVerb(Message<Commit> message)
         // Prepare phase incorporates a read to check the cas condition, so a read sensor is registered in addition to the write sensor
         sensors.registerSensor(context, Type.READ_BYTES);
         sensors.registerSensor(context, Type.WRITE_BYTES);
+        sensors.registerSensor(context, Type.INTERNODE_BYTES);
+        sensors.incrementSensor(context, Type.INTERNODE_BYTES, message.payloadSize(MessagingService.current_version));
```

Comment:
I'm not really familiar with UCU implementation, is it ok to count just the payload and not the whole message ?

Reply:
Yes, it is a decision we made in the [past](https://github.com/datastax/cassandra/pull/1121#discussion_r1602111268)

The TLDR is a\ headers have small foot print b\ they are constant with respect to payload size and c\ it is really not possible to fully calculate the message size and include it in the message itself (chicken and egg problem)